pure3.md

```diff
@@ -0,0 +1,182 @@
+PURE presents a "stop-gap" option. In this configuration, the repository
+verifies the software supply chain of the image and signs an in-toto _summary
+link_ with the keys used to sign either the top level Targets or delegated
+signer role. This way, the key used to sign the summary link metadata is
+securely bootstrapped and can be rotated in case of a compromise.
+
+For each target image, the _custom_ field points to the summary link rather than
+all the in-toto metadata. The client must download and verify the Targets
+metadata for the summary link and, if this passes, download and verify the hash
+and signature on the summary link. Once the authenticity of the summary link is
+established, the client must verify that the hashes in the products field of the
+summary link match those of the image.
+
+The summary link does offer some evidence that the repository performed in-toto
+verification on behalf of the automobile, but it is **not** a sufficiently
+strong replacement for actual verification on the vehicle. A key issue with the
+stop-gap option is that it turns the Image repository into a single point of
+failure and therefore can only be trusted as long as the repository remains
+secure. As such, this model must only be adopted as a stop-gap option until the
+update system can be transitioned to the full model in which in-toto
+verification happens on individual vehicles.
+
+
+## Security Analysis
+
+Scudo is an extension for Uptane that adds in-toto's software supply chain
+security properties. Therefore, its threat model is a combination of the threat
+models of Uptane for the delivery of the software artifacts, and in-toto for the
+development of the image as it passes through the software supply chain.
+
+in-toto is a comprehensive framework that secures each step in the software
+supply chain. Furthermore, it plays well with other systems and tools that
+attempt to secure specific aspects of the software supply chain. For example,
+though in-toto can be used to enforce policies pertaining to the flow of
+artifacts through the supply chain, and can provide proof of the people or bots
+authorized to make changes at each step, it does not intrinsically provide
+mechanisms to store the metadata. Yet, since systems like Grafeas or Sigstore's
+Rekor already have support for in-toto, the repository can instead leverage
+these systems to store the metadata. Both of these systems already have support
+for in-toto. Similarly, in-toto does not provide the information found in a
+Software Bill of Materials (SBOM) in a standardized manner. With the importance
+of SBOMs growing in the past few years due to both the recognition that knowing
+the components in a software artifact is vital to supply chain security and
+because they are at the heart of a number of new regulations and standards,
+in-toto is working to develop capabilities that can express SPDX and CycloneDX
+SBOMs as attestations. This means the distinct security properties of both
```

Comment:
the second half of this paragraph sounds like motivation for using in-toto, and should go near the top of the document

Reply:
I've created a separate Reasoning header for describing why in-toto was selected as the technology to pair with Uptane.